src/utilities/alerter.ts

```diff
@@ -43,73 +56,38 @@ export function createAlerter(config: IConfig, log: Bunyan) {
 		const iAmHome = detectedMacs.has(config.myPortableDevice.mac);
 
 		if (iWasHome && !iAmHome) {
-			alertGoodbye();
+			triggerAlert('departure');
 		} else if (!iWasHome && iAmHome) {
-			alertWelcomeHome();
+			triggerAlert('arrival');
 		} else {
 			const knownPortableDevices = (config.knownPortableDevices || []).filter(kd =>
 				detectedMacs.has(kd.mac)
 			);
 
 			if (iWasHome && iAmHome) {
-				alertDoorOpen();
+				triggerAlert('doorOpen');
 			} else if (!iWasHome && !iAmHome) {
 				log.info({ knownPortableDevices });
 
 				if (knownPortableDevices.length === 0) {
-					alertNobodyHome();
+					triggerAlert('intruder');
 				} else {
-					alertSomeoneHome(knownPortableDevices);
+					triggerAlert('knownDevice', knownPortableDevices);
 				}
 			}
 		}
 	}
 
-	function alertGoodbye() {
-		const email = {
-			...config.emailRecipient,
-			text: 'Goodbye.',
-		};
-
-		alert(email);
-	}
-
-	function alertWelcomeHome() {
-		const email = {
-			...config.emailRecipient,
-			text: 'Welcome home.',
-		};
-
-		alert(email);
-	}
-
-	async function alertDoorOpen() {
-		const email = {
-			...config.emailRecipient,
-			text: 'The door is open.',
-		};
-
-		alert(email);
-	}
-
-	async function alertSomeoneHome(detectedKnownPortableDevices: IDevice[]) {
-		const grammar = detectedKnownPortableDevices.length > 1 ? 'are' : 'is';
-
-		const email = {
-			...config.emailRecipient,
-			text: `${detectedKnownPortableDevices.map(d => d.name).join(', ')} ${grammar} home.`,
-		};
-
-		alert(email);
-	}
+	function triggerAlert(type: AlertType, detectedKnownPortableDevices?: IDevice[]) {
+		const grammar = detectedKnownPortableDevices?.length > 1 ? 'are' : 'is';
 
-	async function alertNobodyHome() {
 		const email = {
-			...config.emailRecipient,
-			text: 'The fortress is in peril.',
+			...config.emailConfig,
+			to: config.emailRecipients[type].join(';'),

```

Comment:
Have you tested that delimiting the recipients with a semicolon works? I'd be happy to if not.

Reply:
I have not. I have used email libraries that allowed this before. I think I remember being able to use `,` or `;`. Emailjs docs are useless though.